docs/guides/error-mitigation-and-suppression-techniques.ipynb

```diff
@@ -120,6 +120,74 @@
     "estimator.options.twirling.shots_per_randomization = 100"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "e5dcab7b",
+   "metadata": {},
+   "source": [
+    "## Probabilistic Error Amplification (PEA)\n",
+    "\n",
+    "PEA is an aplification technique used with [Zero-noise extrapolation(ZNE)](#zne).  The main challenge in successfully implmenting ZNE is to have an accurate model for noise in the expectation value and to amplify the noise in a known way.  For utility-scale experiments, PEA is often the best choice because it can be applied to any circuit that can be run with native noise factor $(\\lambda - 1)$. \n",
```

Comment:
```suggestion
    "PEA is an amplification technique used with [Zero-noise extrapolation(ZNE)](#zne).  The main challenge in successfully implementing ZNE is to have an accurate model for noise in the expectation value and to amplify the noise in a known way.  For utility-scale experiments, PEA is often the best choice because it can be applied to any circuit that can be run with native noise factor $(\\lambda - 1)$. \n",
```

Reply:
Perhaps you could say something about ZNE, such as:

PEA is an amplification technique used with [Zero-noise extrapolation(ZNE)](#zne). Assuming that the "target" circuit of interest suffers noise of strength $\alpha$, ZNE requires implementing many copies of the target circuit, each one suffering a noise whose strength is artificially amplified to values higher than $\alpha$. Finally, the mitigated results are estimated by fitting the noisy data and extrapolating the value at noise strength $\alpha=0$.

One of the main challenges in ZNE is to accurately amplify the noise affecting the target circuit. Many techniques (such as "gate folding") provide easy ways of performing this amplification, but are potentially inaccurate and may lead to incorrect results. PEA is a more sophisticated technique and for utility-scale experiments, it is often the best choice.

Reply:
A few references:
- You can use [this](https://ieeexplore.ieee.org/document/9259940) one for ZNE and for gate folding.
- For "but are potentially inaccurate and may lead to incorrect results" you can use [this](https://www.nature.com/articles/s41567-022-01914-3#Sec7) (see [page 4 of supplementary material](https://arxiv.org/pdf/2108.09197) on the arxiv, where it says that "However, we caution that the method can provide noise amplification results that arbitrarily deviate from the desired result when only slightly more complicated noise models are considered")

Reply:
> * You can use [this](https://ieeexplore.ieee.org/document/9259940) one for ZNE and for gate folding

This requires a subscription.  Is there a free source?

Reply:
You can use the [arXiv url](https://arxiv.org/abs/2005.10921) if you prefer, but I'm not sure what the policy is regarding citations